modules/claims_api/lib/bgs_service/local_bgs.rb

```diff
@@ -270,7 +284,8 @@ def make_request(endpoint:, action:, body:, key: nil) # rubocop:disable Metrics/
         response = log_duration(event: 'connection_post', endpoint:, action:) do
           connection.post("#{Settings.bgs.url}/#{endpoint}", full_body(action:,
```

Comment:
Could you clean up the formatting on this?

Perhaps something like this:
```
body  = full_body(action:, body:, namespace: target_namespace, additional_namespace:)
headers =  {
  'Content-Type' => 'text/xml;charset=UTF-8',
  'Host' => "#{@env}.vba.va.gov",
  'Soapaction' => "\"#{action}\""
}
connection.post("#{Settings.bgs.url}/#{endpoint}", body, headers)
```

Reply:
wil do